sdk/cognitiveservices/Language.TextAnalytics/src/Customizations/TextAnalytics/Models/LanguageInput.cs

```diff
@@ -27,7 +23,7 @@ public LanguageInput()
         /// Initializes a new instance of the LanguageInput class.
         /// </summary>
         /// <param name="id">Unique, non-empty document identifier.</param>
-        public LanguageInput(string countryHint = default(string), string id = default(string), string text = default(string))
+        public LanguageInput(string id = default, string text = default, string countryHint = default)
```

Comment:
Is this a breaking change?

Reply:
It is, which is why this is a new version.

Reply:
Should the package version be bumped, or did that happen in a different change?

Reply:
Looks like I forgot to commit the `.csproj` file. Fixed.